uber/templates/base.html

```diff
@@ -170,7 +170,6 @@
                 <br/><br/>
             </div>
         {% endblock %}
```

Comment:
This is an unrelated fix.  We're already using ``toastr`` to display these messages, so there's no reason to have a separate div.